mixer/cmd/server/cmd/server.go

```diff
@@ -15,426 +15,103 @@
 package cmd
 
 import (
-	"bytes"
-	"crypto/tls"
-	"crypto/x509"
 	_ "expvar" // For /debug/vars registration. Note: temporary, NOT for general use
-	"fmt"
-	"io"
-	"io/ioutil"
-	"net"
-	"net/http"
-	_ "net/http/pprof" // For profiling / performance investigations
```

Comment:
do we want to drop this? pprof is how we've done profiling to date.

Reply:
Removed.